Analytics/Classes/SEGAnalytics.m

```diff
@@ -562,8 +562,16 @@ - (void)refreshSettings
     self.settingsRequest = [self.httpClient settingsForWriteKey:self.configuration.writeKey completionHandler:^(BOOL success, NSDictionary *settings) {
         if (success) {
             [self setCachedSettings:settings];
+        } else {
+            // Hotfix: If settings request fail, fall back to using just Segment integration
```

Comment:
Hotfix seems like a weird comment to leave in. I'd rather put in something like "todo: we need to fix this behaviour"